Using-Mastodon/Apps.md

```diff
@@ -33,8 +33,9 @@ Some people have started working on apps for the Mastodon API. Here is a list of
 
 |App|Source code|Developer(s)|
 |---|-----------|------------|
-|[11t](https://appsto.re/i67Q3LH)|<https://github.com/jeroensmeets/mastodon-app>|[@jeroensmeets@mastodon.social](https://mastodon.social/users/jeroensmeets)|
+|[11t](https://itunes.apple.com/nl/app/11t-for-mastodon/id1180628189)|<https://github.com/jeroensmeets/mastodon-app>|[@jeroensmeets@mastodon.social](https://mastodon.social/users/jeroensmeets)|
```

Comment:
why this change?

Reply:
It the only shortURL, so expand link.
it useless?

Reply:
eeh alright